tdp/cli/commands/plan/run.py

```diff
@@ -11,11 +11,19 @@
 
 @click.command(short_help="Run single TDP operation.")
 @click.argument("operation_names", nargs=-1, required=True)
+@click.option(
+    "--host",
+    envvar="TDP_HOSTS",
+    type=str,
+    multiple=True,
+    help="Hosts where operations are launched. Can be used multiple times.",
+)
 @collections
 @database_dsn
 @vars
 def run(
     operation_names,
+    hosts,
```

Comment:
When I execute tdp plan run --host mdrutel-edge-01 hdfs_jmx-exporter_config, I have this error :

TypeError: run() got an unexpected keyword argument 'host'

We have to modify hosts to host too ^^'

Reply:
Done.